regression/smt2_strings/Makefile

```diff
@@ -0,0 +1,18 @@
+default: tests.log
+
+test:
+	@../test.pl -p -c ../../../src/solvers/smt2_strings
```

Comment:
Should that be `smt2_solver` instead? Not that it would support the theory, but the above doesn't exist at all...